nbgrader/docs/source/user_guide/release/ps1/problem1.ipynb

```diff
@@ -62,6 +36,12 @@
    "execution_count": null,
    "metadata": {
     "deletable": false,
+    "execution": {
```

Comment:
The docs need to be rebuilt in order to exclude these sorts of extraneous metadata. Can you run `python tasks.py docs` from the root of the repo and commit the changes?

(For more details on updating documentation see https://nbgrader.readthedocs.io/en/stable/contributor_guide/documentation.html )

Reply:
We ran `python tasks.py docs` and committed the changes

Reply:
Btw, some new files were created as a result of running this command. Were those expected?
Example: user_guide/autograded/bitdiddle/ps1/problem3.html